spaghetti-core/src/main/ts/tsAstParser.ts

```diff
@@ -88,6 +97,41 @@ class Linter {
                 this.lintError(`missing export * from '${moduleText}' statement`, importDeclaration);
             }
         });
+
+        if (this.hasErrors()) {
+            return null;
+        }
+
+        const replacements: Array<Replacement> = [];
+        statements.forEach((statement) => {
+            const referenceMatch = statement.getFullText().match(/\/\/\/ <reference path=".*" \/>/g);
```

Comment:
use `\s+` ie. `/\/\/\/\s+<reference\s+path=".*"\s+\/>/g);` so you match even if whitespace is different

Reply:
You should find a more general solution like `/\/\/\/[^\n]*/g` because this regex will not match other triple slash directives like `/// <reference types="node" />`